h/static/scripts/directive/annotation.js

```diff
@@ -360,7 +372,7 @@ function AnnotationController(
       // Highlights are always private.
       domainModel.permissions = permissions.private();
       domainModel.$create().then(function() {
-        $rootScope.$emit('annotationCreated', domainModel);
+        $rootScope.$emit(events.ANNOTATION_CREATED, domainModel);
```

Comment:
Not strictly necessary but I don't think it is a problem either, and one less code path to test.